### PR TITLE
Adds Planning service to OpenStack provider

### PR DESCRIPTION
**Pull request by petrblaho**
UPDATE2: modified to work under ruby 1.8.7
UPDATE: content was completely updated. No longer WIP.

This add Planning service to OpenStack provider.

Tuskar [https://github.com/openstack/tuskar/] is Deployment Planning Service for TripleO [https://wiki.openstack.org/wiki/TripleO].
It provides a service building Heat templates for Orchestration of OpenStack deployments (installation of OpenStack).

### CODE REVIEW

**Comment by geemus** [comment]
@mdarby - thoughts?

**Comment by mdarby** [comment]
Seems sane to me so far!

**Comment by geemus** [comment]
Thanks!

File: lib/fog/bin/openstack.rb [hunk 1]
```diff
@@ -20,6 +20,8 @@ def class_for(key)
         Fog::Orchestration::OpenStack
       when :baremetal
         Fog::Baremetal::OpenStack
+      when :planning
+        Fog::Openstack::Planning
```
Comment (Ladas) [inline lib/fog/bin/openstack.rb]: ah I was thinking why it can't find Fog::OpenStack::Compute

you should have there capital S in OpenStack

Comment (petrblaho) [inline lib/fog/bin/openstack.rb]: Yeah, thanks for catching this. Solved.

Comment (petrblaho) [inline lib/fog/bin/openstack.rb]: I had to revert this renaming b/c when I have OpenStack in names it skips all tests for planning service.

**Comment by geemus** [comment]
Awesome, thanks!

**Pull request merged**